src/styles/global.css

```diff
@@ -163,7 +163,7 @@ html {
 
 body {
   margin: 0;
-  padding: 0 1rem;
+  padding: 0 1rem !important;
```

Comment:
Burde nok prøve å unngå important. Ser at det er satt en del important på `body[data-scroll-locked]`. Mulig dette kan være grunnen til at man trenger important her?🤔

Reply:
Klarer ikke finne noe css for det. Ser ikke ut som det er noe vi har lagt til